src/research/pq-verkle.md

```diff
@@ -0,0 +1,84 @@
+Below is a structured summary of the driving motivation behind this research. Verkle has significant benefits over the alternative storage proposals. Nevertheless, it has some drawbacks too like Pedersen Hash speeds and no PQ-security.
+
+Thus, one area of research that is active and stateless consensus is collaborating on together with EF Cryptography team is PQ-Verkle trees.
+You can find more info about the initiative in the following post: ([A short note on Post Quantum Verkle explorations - Execution Layer Research - Ethereum Research](https://ethresear.ch/t/a-short-note-on-post-quantum-verkle-explorations/22001)).
+
+The rest of the page contains a summary of the post and proposal.
+
```

Comment:
Can you also add a table of contents? ([example here](https://github.com/stateless-consensus/eth-stateless-book/blob/main/src/development/mainnet-analysis/tree-shape.md?plain=1#L3-L7))